command/helpers/common.go

```diff
@@ -21,8 +21,11 @@ func GenerateURI(basepath string, endpoint string, serverOverride string) string
         fmt.Fprintf(os.Stdout, "DEBUG [GenerateURI]: basepath->'%s', endpoint->'%s', serverOverride->'%s'\n", basepath, endpoint, serverOverride)
     }
     var uri bytes.Buffer
+    uri.WriteString("http://")
     if serverOverride != "" {
         uri.WriteString(serverOverride)
+    } else if os.Getenv("HASSIO_SERVER") != "" {
+        uri.WriteString(os.Getenv("HASSIO_SERVER"))
```

Comment:
I think we should use the same env variable as other home-assistant project use: `HASSIO`

Reply:
That's Fine